quantum-viz/pyproject.toml

```diff
@@ -48,6 +48,7 @@ pre-commit-hooks = "^4.0.1"
 sphinx-rtd-theme = "^0.5.2"
 sphinx-click = "^3.0.1"
 Pygments = "^2.9.0"
+qiskit-terra = {version = "^0.18.3", extras = ["qiskit"]}
```

Comment:
I think this might be causing the failure in the CI/CD:
```
TypeError: slice indices must be integers or None or have an __index__ method
```

Reply:
Hi, so #45 suggests that the CI fails on the main branch without any further changes.
I think that we should first identify where the issue is, but I do not think it's related to this PR.

Speaking about the pre-commit session alone (`nox -s pre-commit`) - it passes for me locally both on the main branch and on this branch.

Reply:
@jond01, could you please link the build step that failed in #45 so we can investigate? I only see green checkmarks but perhaps I missed something. Thanks!

Reply:
This issue might be related: https://github.com/cjolowicz/cookiecutter-hypermodern-python/issues/1050

Reply:
Most likely, it was a transient issue with the github action or pip - now it passes without any change :)

Probably irrelevant now, but this is the pre-commit run that failed:
https://github.com/jond01/quantum-viz.js/runs/4175974639?check_suite_focus=true

Thanks for updating